.github/workflows/ci.yml

```diff
@@ -188,17 +188,14 @@ jobs:
       - uses: ./.github/actions/setup
         with:
           cache: restore
-      - name: Maven Package
+      - name: Maven Install
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          $MAVEN clean package ${MAVEN_FAST_INSTALL} ${MAVEN_GIB} -pl '!:trino-docs,!:trino-server,!:trino-server-rpm'
+          $MAVEN clean install ${MAVEN_FAST_INSTALL} ${MAVEN_GIB} -pl '!:trino-docs,!:trino-server,!:trino-server-rpm'
       - name: Error Prone Checks
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          # Run Error Prone on one module with a retry to ensure all runtime dependencies are fetched
-          $MAVEN ${MAVEN_TEST} -T 1C clean verify -DskipTests -P gib,errorprone-compiler -am -pl ':trino-spi'
-          # The main Error Prone run
-          $MAVEN ${MAVEN_TEST} -T 1C clean verify -DskipTests -P gib,errorprone-compiler \
+          $MAVEN ${MAVEN_TEST} -T 1C clean verify -DskipTests ${MAVEN_GIB} -Dgib.buildUpstream=never -Dgib.buildDownstream=never -P errorprone-compiler \
```

Comment:
> Errorprone is only needed to check modules with changes, not all
impacted modules, that is downstream dependencies.

i think that's an assumption that's not necessarily always true. a legit change in affected module can cause error-prone failure in a downstream

**for example**, a field change from int to long could cause NarrowingCompoundAssignment failure in eg base-jdbc

Reply:
Done!